command.go

```diff
@@ -241,6 +241,46 @@ func getMPubBody(bodies [][]byte) (*bytes.Buffer, error) {
 	return buf, nil
 }
 
+func getMPubBodyWithJsonExt(extList []*MsgExt, bodies [][]byte) (*bytes.Buffer, error) {
+	num := uint32(len(bodies))
+	jsonExtBytesList := make([][]byte, 0)
```

Comment:
Use `jsonExtBytesList := make([][]byte, 0, len(bodies))` to get better performance